db/dcrpg/upgrades.go

```diff
@@ -455,6 +482,16 @@ func (pgb *ChainDB) handleUpgrades(client *rpcutils.BlockGate,
 	case blocksRemoveRootColumns:
 		tableReady, err = deleteRootsColumns(pgb.db)
 		tableName, upgradeTypeStr = "blocks", "remove columns: extra_data, merkle_root, stake_root, final_state"
+	case agendasTableReorgUpdate:
+		i = 4000
```

Comment:
Please document any magic number choices.

Reply:
Is that correct for all chains?

Reply:
I think this should be set to stakevalidationheight so it works on testnet also.